orca-peering/src/main/kotlin/com/netflix/spinnaker/orca/peering/PeeringAgent.kt

```diff
@@ -76,7 +75,7 @@ class PeeringAgent(
   override fun tick() {
     if (dynamicConfigService.isEnabled("pollers.peering", true) &&
       dynamicConfigService.isEnabled("pollers.peering.$peeredId", true)) {
-      peeringMetrics.recordOverallLag() {
+      peeringMetrics.recordOverallLag {
```

Comment:
is it the same with and without the parens? I'm confused

Reply:
yes, i guess spotless did it.. but yeah, i guess that's kotlin convention: https://kotlinlang.org/docs/reference/lambdas.html#passing-a-lambda-to-the-last-parameter